src/storage/swap.cc

```diff
@@ -170,7 +170,7 @@ void Swap::DelAddr(handle_id_t handle_id) {
   auto info = swap_info_.at(handle_id);
   if (info->device_id != -1) {
     memory_history_->PutRecord(handle_id, info->device_id, MemHistory::DEL_ADDR, info->size);
-    if (swappable_handles_[info->device_id].find(handle_id) 
```

Comment:
Were there some formatting errors at some ends of lines? I notice lots of changes at ends of line like this one